cves/kernel/CVE-2020-15393.yml

```diff
@@ -55,7 +55,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: A function for testing a USB disconnect routine failed to free up a chunk of memory space that had been reserved by it before finishing execution.
```

Comment:
consider splitting to more lines for readability

Reply:
Looked up how to do this, will do.